dev/pastetools/getclipboard.html

```diff
@@ -0,0 +1,309 @@
+<!DOCTYPE html>
+<!--
+Copyright (c) 2003-2020, CKSource - Frederico Knabben. All rights reserved.
+For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+-->
+<html>
+<head>
+	<meta charset="UTF-8">
+	<title>Get Clipboard HTML and RTF</title>
+	<style>
+
+		textarea {
+			border: 1px solid #808080;
+			float: left;
+			height: 200px;
+			width: 100%;
+			overflow: auto;
+			margin-bottom: 20px;;
+		}
+		label {
+			display: block;
+			line-height: 2;
+		}
+		span {
+			background-color: #88FFAA;
+		}
```

Comment:
Using styles in which such small specificity may also change editor styling. It's e.g. visible when opening emoji completion where items have a green background.